packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTabs/CWTab.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWTag } from 'views/components/component_kit/new_designs/cw_tag';
+
+import './CWTab.scss';
+
+interface CWTabProps {
+  label: string;
+  isSelected: boolean;
+  onClick: () => void;
+  isDisabled?: boolean;
+  showTag?: boolean;
+}
+
+const CWTab = ({
+  label,
+  isDisabled,
+  isSelected,
+  onClick,
+  showTag,
+}: CWTabProps) => {
+  return (
+    <button
```

Comment:
I suggest we set button type to "type=button", else if used inside a form it will submit the form (since the default type is "submit").

Reply:
good call, updated